cosmo_tester/test_suites/cli/__init__.py

```diff
@@ -57,6 +63,85 @@ def _test_cfy_install(run, example, paths, logger):
     example.check_files()
 
 
+def _set_ssh_in_profile(run, example, paths):
+    run(
+        '{cfy} profiles set --ssh-user {ssh_user} --ssh-key {ssh_key}'.format(
+            cfy=paths['cfy'],
+            ssh_user=example.manager.username,
+            ssh_key=paths['ssh_key'],
+        )
+    )
+
+
+def _test_cfy_logs(run, cli_host, example, paths, logger):
+    _set_ssh_in_profile(run, example, paths)
+
+    linebreak = '\\r\\n' if 'C:\\' in paths['cfy'] else '\n'
+    logger.info('Testing `cfy logs download`')
+    logs_dump_filepath = str(run(
+        '{cfy} logs download '.format(cfy=paths['cfy'])
+    ).stdout).split('archive to:')[1].split(linebreak)[0]
+
+    # Not working on windows!
```

Comment:
So let's make cfy logs download retrieve a zip file.

Reply:
we need zip installed on the manager for that. Currently it isn't

Reply:
OK, in case anyone consults this later: We've had a bunch of back and forth about the windows side of this, and the upshot is that it's going to require some more work to handle windows with this so we're ignoring it for now.